raft/raft.hh

```diff
@@ -450,12 +442,39 @@ struct entry_id {
     index_t idx;
 };
 
+// The execute_add_entry/execute_modify_config methods can return this error to signal
+// that the request should be retried.
```

Comment:
Lets specify that the exception is only used internally for entry/config forwarding and should not be leaking to a user.